src/ProjectOrigin.WalletSystem.IntegrationTests/TransferCertificateTests.cs

```diff
@@ -70,7 +68,7 @@ public async Task Transfer_SingleSlice_LocalWallet()
         await client.TransferCertificateAsync(request, senderHeader);
 
         //Assert
-        await WaitForCertCount(certId, 4);
+        await WaitForCertCount(certId, 3);
```

Comment:
Why are the number of certificates to wait for changed? This is also the case for other tests in this file

Reply:
Since before they also counted those which are in the transferred_slices table, but now since the tables are split it only counts wallet_slices